lib/pg_online_schema_change/orchestrate.rb

```diff
@@ -16,6 +18,13 @@ def setup!(options)
         # install functions
         Query.run(client.connection, FUNC_FIX_SERIAL_SEQUENCE)
         Query.run(client.connection, FUNC_CREATE_TABLE_ALL)
+
+        # Set this early on to ensure their creation and cleanup (unexpected)
+        # happens at all times. IOW, the calls from Store.get always return
+        # the same value.
+        Store.set(:old_primary_table, "pgosc_op_table_#{client.table}")
```

Comment:
We should make sure to set the maximum length of these table names to 63. In fact, all identifiers in postgresql are limited to `NAMEDATALEN - 1` or 63.

Reply:
yep def! Since we change the sequence name too (so its unique between primary and shadow table), keeping it short is nice. 

Just to future proof, i added a spec. They are between 20/21 char len.